code/datums/status_effects/debuffs.dm

```diff
@@ -402,6 +402,7 @@
 	id = "sleeping"
 	tick_interval = 2 SECONDS
 	needs_update_stat = TRUE
+	var/voluntary = FALSE
```

Comment:
would you add a comment to what this does?

Reply:
Done